### DISCUSSION TITLE
New features after login?

### DISCUSSION THREAD

**Discussion by bobhuf** [responsively-org/responsively-app#1228] [General]
What are some of the "cool new features" that will be unlocked by registering and logging in?

![Scherm­afbeelding 2024-05-08 om 13 28 09](https://github.com/responsively-org/responsively-app/assets/4223412/c7693283-fc12-477a-a643-38fc38ba5fbb)

**Comment by manojVivek** [comment]
It is in the works to allow syncing of Preview Suits (the devices list that the user has added to their various suits) across installations. 

Would love to hear if you have any suggestions.

**Reply by bobhuf** [reply]
Then at the moment there is no incentive for me to register _except_ to remove the above screen. I wouldn't mind registering with email just for that but connecting my google or github account is a bit much.

**Comment by technigit** [comment]
I was also puzzled about this.  I got a screen to choose an account to continue to something like ekwgibberish.supabase.co and that made me feel a bit nervous.

My suggestions:

1) It would be a very good idea to change that address to something that doesn't look suspiciously like a hacking attempt.  (I did some quick research and found out that supabase.co is a legitimate database provider along the lines of Firebase.  Surely they provide a way to set up a better-looking address?)

2) Adding a "don't ask me again" checkbox would be excellent.  (Of course, also provide a way for the user to undo that action and unlock some cool features in the app.)

**Reply by manojVivek** [reply]
@technigit Thanks for letting us know your concern, we have fixed the redirect URL to a `responsively.app` subdomain now.
